src/main/java/uk/gov/ons/ctp/response/sample/mapper/SampleUnitMapper.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.ons.ctp.response.sample.mapper;
+
+import ma.glasnost.orika.CustomMapper;
+import ma.glasnost.orika.MappingContext;
+import uk.gov.ons.ctp.response.sample.domain.model.SampleUnit;
+
+public class SampleUnitMapper extends CustomMapper<SampleUnit, uk.gov.ons.ctp.response.sampleunit.definition.SampleUnit> {
+
+    @Override
+    public void mapAtoB(SampleUnit sampleUnit, uk.gov.ons.ctp.response.sampleunit.definition.SampleUnit sampleUnit2, MappingContext context) {
```

Comment:
maybe importing 'uk.gov.ons.ctp.response.sampleunit.definition.SampleUnit' at the top instead will avoid the long 'uk.gov.ons.ctp.response.sampleunit.definition.SampleUnit.SampleAttributes()' again